src/datasets/search/client_ltv_normalized/intro.md

```diff
@@ -0,0 +1,46 @@
+`client_ltv_normalized` is designed to enable relative user value estimates based on their past and expected search and ad click behavior. This behavior is revenue-generating for Firefox.
```

Comment:
s/`client_ltv_normalized`/`client_ltv_v1`. We can specify in the bottom part that the `revenue` project contains the non-normalized values, if someone needs them.